_posts/2017-04-30-React-Native-for-iOS-devs.markdown

```diff
@@ -0,0 +1,671 @@
+---
+layout: epic
+title: Intro to React Native for an iOS Developer
+date: 2017-05-30
+categories: [Technology, emission, react-native, react, javascript]
+author: orta
+series: React Native at Artsy
+css: what-is-react-native
+---
+
+React Native is a new native library that vastly changes the way in which you can create applications. The majority of the information and tutorials on the subject come from the angle of _"you are a web developer, and want to do native"_.
```

Comment:
For the iOS ecosystem, 2 years is like _"maybe we should consider adopting this new fangled thing"_ even when it comes from Apple. I feel like it's still safe from _my_ perspective to call it that.